sdk/Eversign/Client.php

```diff
@@ -572,6 +572,32 @@ public function downloadDocumentToPath(Document $document, $path, $auditTrail =
         return $request->startRequest();
      }
 
+    public function downloadFinalDocument(Document $document, $auditTrail = 0, $onlyUrl = false, $documentId = "") {
+
+        $parameters = [
+            "business_id" => $this->selectedBusiness->getBusinessId(),
+            "document_hash" => $document->getDocumentHash(),
+            "audit_trail" => $auditTrail,
+            "document_id" => $documentId,
+            "url_only" => $onlyUrl
+        ];
+
+        $payLoad = $onlyUrl ? null : ["stream" => false];
+
+        $request = new ApiRequest(
+            "GET",
+            $this->accessKey,
+            'download_final_document',
```

Comment:
There is already a constant for this string, we should use it `Config::DOCUMENT_FINAL_URL`

Reply:
Oh true! I missed it, thank you